sources/packages/api/src/services/institution-location/institution-location.service.ts

```diff
@@ -144,4 +140,23 @@ export class InstitutionLocationService extends RecordDataModelService<Instituti
   ): Promise<InstitutionLocation[]> {
     return this.repo.findByIds(locationIds);
   }
+
+  /**
+   * Gets all locations ids for a particular institution.
+   * This method is used during the login process and should be
+   * as lightweight as possible. Do not expand this query unless
+   * it is related to login/authorization process.
+   * @param institutionId institution id.
+   * @returns institution locations ids.
+   */
+  async getInstitutionLocationsIds(institutionId: number): Promise<number[]> {
+    const allLocations = await this.repo
+      .createQueryBuilder("locations")
```

Comment:
table name is `institution_location` right? am I missing something?

Reply:
This is not a table name, it is the alias to execute the query. IMO using the full table name as an alias doesn't make sense and it also makes the code longer and hard to read.
Please let me know if it makes sense.